auxloss/tests/test_aux_power_consumption.py

```diff
@@ -38,25 +38,25 @@ def test_expected_headers_stored():
         assert("Max Power (W)" in headers and "Typical Power (W)" in headers)
 
 def test_calculate_instantaneous_power_string_names_only():
-    assert(auxpc.calculate_instantaneous_power({"components": ['Center Console']}) == 4.4)
-    assert(auxpc.calculate_instantaneous_power({"components": ["Horn", "Motor Interface", "Solar Master"]}) == 0 + 2.4 + 4.0)
+    assert(auxpc.calculate_instantaneous_power({"components": ['Center Console']}) == 16.54)
```

Comment:
I think it's worth identifying here which components are unknowns according to the new budget and removing them from the upper tests. That logic is tested in the `test_calculate_instantaneous_power_ignores_unknown_components`. We can replace them with new components in the power budget instead and update the expected values 👍

Reply:
Hi Emma, can you explain how to approach this? We are not sure of what to do from the comment.